test/processCommand_test.js

```diff
@@ -0,0 +1,140 @@
+goog.require('helper');
+
+describe('The processCommand function', function() {
```

Comment:
nit: please add quotes/ticks around processCommand, and add the trailing underscore.